### ISSUE TITLE
Hyperparameter for ImageNet

### ISSUE THREAD

**Issue by chakkritte** [lgcnsai/PS-KD-Pytorch#4]
Hi @lgcnsai @ByeongMoonJi,

Thanks for your excellent work!

I have seen a hyperparameter for ImageNet in the main paper, which is a batch size of 256, while in supplementary paper used 512.

- What exactly batch size value that you used in Table 4? 
- One more thing, could you provide me the alpha value for ImageNet dataset that used in Table 4?
- Lastly, could you upload the ResNet-152 pre-trained model (Baseline, with LS,  with  CS, with  TF, and with PS-KD) to available on GitHub ?

Best Regards,
Chakkrit

**Comment by MsterDC** [comment]
Hi, I met a similar problem.
Recently, I trained on ResNet-50 instead of ResNet-152. The hyperparameters suggested alpha_T=0.3 and batch size=256 instead of 512 according to the supplementary material, but the results were very poor.  
When training to 50+epoch, the accuracy rate is only about 20%, I wonder if anyone knows why? @lgcnsai @ByeongMoonJi

**Comment by lgcnsai** [comment]
Hi, @chakkritte 
To find out the optimal hyperparameter of PS-KD, we performed a random search of hyperparameters over five trials with ResNet-152 for a fair comparison. In this process, We needed to set the minibatch size to 512, and then, at imageNet experiments in table4, we set the minibatch size to 256 and optimal alpha_T.

Hope that helps

**Comment by lgcnsai** [comment]
Hi, @MsterDC 

For checking out our code and performance as you mentioned, we trained again PS-KD on ResNet-50 instead of ResNet-152. (set to the same as ImageNet experiments in the main manuscript)

Please see the result,

- resnet-152  `net = ResNet(dataset = 'imagenet', depth=152, num_classes=1000, bottleneck=True)`
6 epoch : val top1-acc 44.99
- resnet-50  `net = ResNet(dataset = 'imagenet', depth=50, num_classes=1000, bottleneck=True)`
7 epoch : val top1-acc 47.31

Hope that helps
